extensions/fuel-it/.env.example

```diff
@@ -0,0 +1,2 @@
+OPENAI_API_KEY=
+FUEL_WALLET_PRIVATE_KEY=
```

Comment:
File should be removed 😊

Reply:
I already removed that file. I am not able to see `.env.example` file in my local editor
![image](https://github.com/user-attachments/assets/de943bce-10ac-41e1-8e1f-22afa9aacbef)

Reply:
its not there in my git version tracking as well